src/ci/docker/disabled/wasm32/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN sh /scripts/dumb-init.sh
 # emscripten
 COPY scripts/emscripten.sh /scripts/
 RUN bash /scripts/emscripten.sh
-COPY wasm32/node.sh /usr/local/bin/node
+COPY disabled/wasm32/node.sh /usr/local/bin/node
```

Comment:
This would have to be changed if the builder was ever moved out of the docker/disabled directory, but for now lets the builder be run locally without modification.